client/src/components/shared/ButtonAddToAllocate/ButtonAddToAllocate.tsx

```diff
@@ -28,6 +28,9 @@ const ButtonAddToAllocate: FC<ButtonAddToAllocateProps> = ({
   const [isTooltipClicked, setIsTooltipClicked] = useState(false);
   const [isTooltipVisible, setIsTooltipVisible] = useState(false);
   const tooltipText = useMemo(() => {
+    if (isArchivedProject) {
```

Comment:
`Donated` should show only when `isArchivedProject` and `isAllocatedTo`.

Reply:
`ButtonAddToAllocate` is rendered for archived project only if user allocated funds to the project. I'm ok with adding double check :)

Reply:
@aziolek done